tests/test_converters.py

```diff
@@ -0,0 +1,241 @@
+import pytest
+
+import ujt.converters
+from generated.graph_structures_pb2 import (SLI, Client, Dependency, Node,
+                                            NodeType, UserJourney)
+
+
+@pytest.fixture
+def patch_path():
+    return "ujt.converters"
+
+
+def test_cytoscape_elements_from_nodes():
+    node_name_message_map = {
+        "Service1":
+            Node(node_type=NodeType.NODETYPE_SERVICE,
+                 name="Service1",
+                 child_names=["Service1.Endpoint1", "Service1.Endpoint2"],
+                 slis=[
+                     SLI(node_name="Service1",
+                         sli_value=.5,
+                         **ujt.generate_data.SLO_BOUNDS)
+                 ]),
+        "Service2":
+            Node(node_type=NodeType.NODETYPE_SERVICE,
+                 name="Service2",
```

Comment:
It's generally better to define string constants for things like this; it keeps you from getting bugs due to typos, and makes relationships like "all service one endpoints start with Service1." more explicit.